spec/shoryuken/polling_spec.rb

```diff
@@ -0,0 +1,63 @@
+require 'spec_helper'
+require 'shoryuken/polling'
+
+describe Shoryuken::Polling do
+  let(:queue1) { "shoryuken" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.